src/lib/components/Stepper.react.js

```diff
@@ -0,0 +1,153 @@
+import { Stepper as MantineStepper } from '@mantine/core';
+import PropTypes from "prop-types";
+import React from "react";
+import { renderDashComponents } from "dash-extensions-js";
+import { omit } from "ramda";
+
+
+const Stepper = (props) => {
+
+    const { children, class_name } = props;
+    console.log("log child", children)
+    return (
+        <MantineStepper
+            {...omit(["children", "class_name", "setProps"], props)}
+            className={class_name}
+        >
+            {React.Children.map(children, (child, index) => {
+
+                const childProps = child.props._dashprivate_layout.props;
+                const childType = child.props._dashprivate_layout.type;
+
+                const renderedProps = renderDashComponents(
+                    omit(["children"], childProps),
+                    ["completedIcon", "description", "icon", "label", "progressIcon",]
+                );
+
+                console.log(renderedProps, index)
+                if (childType === "StepperStep") {
+                    return (
+                        <MantineStepper.Step
+                            {...omit(["children", "completedIcon",
+                                "description", "icon", "label",
+                                "progressIcon"], childProps)}
+                            {...renderedProps}
+                            key={index}
+                        >
+                            {childProps.children}
+                        </MantineStepper.Step>
+                    );
+                } else if (childType === "StepperCompleted") {
+                    return (
+                        <MantineStepper.Completed>
+                            {child}
+                        </MantineStepper.Completed>
+                    );
+                }
+            })}
+        </MantineStepper>
+    )
+
+}
+
+Stepper.displayName = "Stepper";
+
+Stepper.defaultProps = {};
+
+Stepper.propTypes = {
+    /**
+    * Active step index
+    */
+    active: PropTypes.number,
+    /**
```

Comment:
Please ensure proper spacing and formatting.

```suggestion
    
    /**
```